assembl/static2/js/app/reducers/adminReducer/index.js

```diff
@@ -198,6 +198,17 @@ export const discussionLanguagePreferencesHasChanged: DiscussionLanguagePreferen
   }
 };
 
+type ToggleLanguageMenu = boolean;
+type ToggleLanguageMenuReducer = (ToggleLanguageMenu, ReduxAction<Action>) => ToggleLanguageMenu;
+export const toggleLanguageMenu: ToggleLanguageMenuReducer = (state = false, action) => {
```

Comment:
I think it is really a strange name for a reducer, maybe use something like `languageMenuIsVisible` or `displayLanguageMenu`

Reply:
I agree. Let's do that...